SECURITY.md

```diff
@@ -0,0 +1,174 @@
+[Security Team](#security-team) about the vulnerability so they may start the
+patch, release, and communication process.
+
+If possible the Security Team will ask the person making the public report if
+the issue can be handled via a private disclosure process. If the reporter
+denies the request, the Security Team will move swiftly with the fix and
+release process. In extreme cases you can ask GitHub to delete the issue but
+this generally isn't necessary and is unlikely to make a public disclosure less
+damaging.
+
+### Security Releases
+
+Once a fix is available it will be released and announced via the project on
+GitHub. Security releases will announced and clearly marked as a security
+release and include information on which vulnerabilities were fixed. As much as
+possible this announcement should be actionable, and include any mitigating
+steps users can take prior to upgrading to a fixed version.
+
+Fixes will be applied in patch releases to all [supported
+versions](#supported-versions).
+
+### Severity
+
+The [Security Team](#security-team) evaluates vulnerability severity on a
+case-by-case basis, guided by [CVSS 3.1](https://www.first.org/cvss/v3.1/specification-document).
+
+## Security Posture
+
+We aim to reduce the number of security issues through several general
+security-concious development practices including the use of unit-tests,
+end-to-end (e2e) tests, static and dynamic analysis tools, and use of
+memory-safe languages.
+
+We aim to fix issues discovered by analysis tools as quickly as possible. We
+prefer to add these tools to "pre-submit" checks on PRs so that issues are
+never added to the code in the first place.
+
+In general, we follow the following security-concious practices during
+development (This is not an exhaustive list).
+
+- All PRs are reviewed by at least one [CODEOWNER](./CODEOWNERS).
+- All unit and linter pre-submit tests must pass before a PRs is merged. See
+  the [Pre-submits and Unit Tests](./CONTRIBUTING.md#pre-submits-and-unit-tests)
+  section of the Contributor Guide for more information.
+- All releases include no known e2e test failures. See
+  [RELEASE.md](./RELEASE.md) for info on the release process. See the
+  [End to End (e2e) Tests](./CONTRIBUTING.md#end-to-end-e2e-tests) section of
+  the Contributor Guide for more information on e2e tests.
+- We refrain from using memory-unsafe languages (e.g. C, C++) or memory-unsafe
+  use of languages that are memory-safe by default (e.g. the Go
+  [unsafe](https://pkg.go.dev/unsafe) package).
+
+## Security Team
+
+The Security Team is responsible for the overall security of the
+project and for reviewing reported vulnerabilities. Each member is familiar
+with designing secure software, security issues related to CI/CD, GitHub
+actions and build provenance.
+
+You can view current team membership here:
+[slsa-framework/slsa-github-generator-security](https://github.com/orgs/slsa-framework/teams/slsa-github-generator-security/members)
```

Comment:
I created this team and I intend to initially add the 4 current repo admins. However, I wanted input on this PR before spamming invites to the GitHub team.

Reply:
I'm fine with the team.

Reply:
AIUI teams and their members are only visible to members of the organisation, so that URL won't be useful to most folks reading this document.

https://docs.github.com/en/organizations/organizing-members-into-teams/changing-team-visibility

Reply:
hrm, well that means that it won't be useful for this purpose. I guess we can just add names here.

Reply:
I added a list of names since a team can't be viewed externally to the org.